.github/workflows/deploy-packages.yml

```diff
@@ -7,7 +7,7 @@ concurrency:
 on:
   workflow_dispatch:
   workflow_run:
-    workflows: [Release]
+    workflows: [Promote]
```

Comment:
Ah yeah because it needs the packages to be on latest ... this makes total sense. Thanks for fixing this!